MediaBrowser.Controller/Net/IAuthService.cs

```diff
@@ -1,9 +1,12 @@
+using MediaBrowser.Controller.Entities;
 using MediaBrowser.Model.Services;
+using Microsoft.AspNetCore.Http;
 
 namespace MediaBrowser.Controller.Net
 {
     public interface IAuthService
     {
         void Authenticate(IRequest request, IAuthenticationAttributes authAttribtues);
+        User Authenticate(HttpRequest request, IAuthenticationAttributes authAttribtues);
```

Comment:
why two methods?

Reply:
The new one converts an asp.net httprequest to a websocketsharprequest and returns the user. It is only for compatibility.